charts/vault-secrets-operator/Chart.yaml

```diff
@@ -7,4 +7,4 @@ maintainers:
   - name: Rico Berger
     url: https://ricoberger.de
 name: vault-secrets-operator
-version: 1.3.0
+version: 1.3.1
```

Comment:
Can you revert this change please. The version bump is handled via the Makefile.

Reply:
done!